LBH.AdultSocialCare.Api.Tests/V1/DataGenerators/CarePackageGenerator.cs

```diff
@@ -20,7 +19,12 @@ public async Task<CarePackage> CreatePackage(PackageType type)
         {
             var serviceUser = await _context.Clients.FirstOrDefaultAsync().ConfigureAwait(false);
 
-            var package = TestDataHelper.CreateCarePackage(packageType: type, serviceUserId: serviceUser.Id, status: PackageStatus.New);
```

Comment:
To Duncan: had to rollback it cause it fails on random supplier, so would need to add another parameter for supplier... So method call would be too complicated

Let's keep it without randomization, since in integration tests we need well-known values, unlike unit tests where we not limited by DB constrains.